src/G2/Execution/Reducer.hs

```diff
@@ -428,6 +430,31 @@ stdRed share symb_func_eval solver simplifier =
                             return (if r == RuleIdentity then Finished else InProgress, s', b')
                         )
 
+nonRedPCTemplates :: Monad m => Reducer m () t
```

Comment:
Let's add at least a short comment to document this function. Use Haddock syntax:
`-- | Haddock Comment`
so it will show up in documentation.